index.html

```diff
@@ -214,5 +214,52 @@ <h1>About Me</h1>
         </div>
       </section>
     </main>
+    <footer>
+      <section class="get-in-touch">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+      </section>
+      <form action="https://formspree.io/f/meqdgwge" method="POST">
+        <div class="form-group">
+          <input
+            type="text"
+            name="name"
+            placeholder="Full Name"
+            maxlength="30"
+            required
+          />
+          <input type="email" name="email" placeholder="Email" required />
+          <textarea
+            cols="33"
+            rows="3"
+            name="message"
+            placeholder="Enter text here"
+            required
+          ></textarea>
+        </div>
+        <button type="submit">Get in touch</button>
+        <nav class="contact-me">
+          <a class="logo1" href="https://github.com/amexabee"
+            ><i class="fa fa-github"></i
+          ></a>
+          <a
+            class="logo2"
+            href="https://www.linkedin.com/in/amanuel-galema-56900024a/"
+            ><i class="fa fa-linkedin"></i
+          ></a>
+          <a class="logo3" href="https://angel.co/"
+            ><i class="fa fa-angellist"></i
+          ></a>
+          <a class="logo4" href="https://twitter.com/fountfillwblood"
+            ><i class="fa fa-twitter"></i
+          ></a>
+          <a class="logo5" href="https://medium.com/"
+            ><i class="fa fa-medium"></i
+          ></a>
+        </nav>
```

Comment:
- [ ] Kindly add the line below icons as it shows in the picture below it will make your design to stick to the Figma design as required by the project 

![image](https://user-images.githubusercontent.com/95297251/192661861-80d8ca24-448b-4594-824b-e8fdd38b7593.png)